itests/dummydeal_test.go

```diff
@@ -84,4 +84,11 @@ func TestDummydeal(t *testing.T) {
 	require.False(t, res.Accepted)
 	require.Contains(t, res.Reason, "storage price per epoch less than asking price")
 	log.Debugw("got response from MarketDummyDeal", "res", spew.Sdump(res))
+
+	// deal fails because signature is invalid
+	res, err = f.makeDummyDeal(dealUuid, carFilepath, rootCid, server.URL+"/"+filepath.Base(carFilepath), false, abi.NewTokenAmount(2000000), withInvalidSignature())
+	require.NoError(t, err)
+	require.False(t, res.Accepted)
+	require.Contains(t, res.Reason, "error verifying signature")
+	log.Debugw("got response from MarketDummyDeal", "res", spew.Sdump(res))
```

Comment:
I think we don't really need the invalid signature and min ask tests at the dummy deal level if we're already testing these things in the provider unit test

Reply:
@dirkmc Dosen't hurt to have it for a sanity check as these are two important validations and these finish quickly as we don't wait for these to be sealed since they get rejected right off the bat.

Reply:
I think I'd rather keep all the validation testing in one place. Someone looking at this code later might think they should add validation testing code here. In general I favour having the minimal amount of tests to keep the code maintainable.

Reply:
Okay, done.